pyflakes/checker.py

```diff
@@ -1208,6 +1219,7 @@ def handleNodeLoad(self, node):
 
             binding = scope.get(name, None)
             if isinstance(binding, Annotation) and not self._in_postponed_annotation:
+                scope[name].used = True
```

Comment:
i only really need this to be truthy, but should i define it as a tuple, like in the `try` below?